src/main/java/org/cyclonedx/model/Dependency.java

```diff
@@ -36,6 +37,11 @@ public class Dependency extends BomReference {
     @JacksonXmlProperty(localName = "dependency")
     private List<Dependency> dependencies;
 
+    @VersionFilter(Version.VERSION_16)
+    @JsonProperty("provides")
+    @JacksonXmlProperty(localName = "provides")
+    private List<Dependency> provides;
```

Comment:
It should be `private List<BomReference> provides` the provide is linked to a BomRef not to a dependency based on the spec